ml_model/src/main/java/edu/wm/sealab/featureextraction/Parser.java

```diff
@@ -75,6 +91,12 @@ public static void main(String[] args) {
                     new SyntacticFeatureExtractor(featureVisitor.getFeatures());
                 Features features = sfe.extract(cuNoComm.toString());
 
+                 String line;
+                line = sc.next();
```

Comment:
How the records from each line match with the order of the snippets? I suggest to use a CSV file reader to match records.